app/assets/stylesheets/application.scss

```diff
@@ -325,14 +325,22 @@ ol.with-bullets {
 .fyst-home-content {
   flex: 1;
   padding: 15px;
+  padding-top: 45px;
 }
 .fyst-home-image {
   flex-shrink: 0;
-  width: 25%;
+  width: auto;
   height: auto;
   padding: 15px;
-  max-width: 300px;
-  max-height: 280px;
+  padding-top: 45px;
   align-self: flex-start;
 }
 
+/* Media query for screens smaller than 768px (typical mobile devices) */
```

Comment:
We have a media query above with "tablet-up" - should we use that instead?

Reply:
Good catch, thanks!

Reply:
$tablet-up is configured to hide whenever the screen is 600px and lower so it looks off in the 6 hundred some range screen sizes

![Screen Shot 2024-02-08 at 4 53 50 PM](https://github.com/codeforamerica/vita-min/assets/2385700/161d98fe-c2c1-4d33-b240-245f3a7c485d)